changelog.md

```diff
@@ -1,5 +1,9 @@
 # Gro changelog
 
+## unreleased
```

Comment:
do you know what the this release version this will be in? Any reason to not just preemptively set it here?

Reply:
ya it'll be 0.1.6, I did that as a workflow thing copying another project I saw, but my reasoning was flimsy - basically didn't want to confuse that 0.1.6 could already be released. But that's not really an issue for us. Certainly easier to just put the version that it should be and not worry about changing it in the future. I'll just put the target version unless you have any thoughts?

Reply:
Hmmm, there's got to be some sort of best practice for updating the version in the changelog and releasing at the same time but it escapes me at the moment. But I think the confusion is better than forgetting to do another PR to update the version on release.

Reply:
Good point, I agree about the confusion is the better of the tradeoffs. Sounds like something we should eventually automate, probably trivial to do in `preversion`, maybe with a simple task.